main.js

```diff
@@ -385,17 +386,15 @@ class AlphaEss extends utils.Adapter {
 
                 if (this.config.enableEnergydata) {
                     // We delay the first start for 3 seconds
-                    const _this = this;
-                    this.settingsDataTimeoutHandle = setTimeout(function () { _this.fetchEnergyData(); }, 3000);
+                    this.settingsDataTimeoutHandle = setTimeout(() => this.fetchEnergyData(), 3000);
```

Comment:
Why you use this call after 3s" at all here?  if you add an await in L381 then a new "fetch" would be exected after the first finished (if you just want to prevent calls "in parallel")

same for the 6s delay below

Reply:
Updated to 1.2.2

Reply:
I think it is important that the calls are executed sequencially ... in fact you will see if a delay of several seconds is really needd, but normally not.

Reply:
OK, I am convinced. :-)